app/index.html

```diff
@@ -17,6 +17,7 @@
         <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
         <!-- build:css(.tmp) styles/main.css -->
         <link rel="stylesheet" href="styles/main.css">
```

Comment:
@mkayan to explore ways to import selectize to main.scss file rather than making extra http request for it.